CHANGELOG.md

```diff
@@ -25,6 +26,7 @@ The changes are relative to the previous release, unless the baseline is specifi
 * Forbid encoding with AVIF_MATRIX_COEFFICIENTS_IDENTITY and
   AVIF_PIXEL_FORMAT_YUV400 to be AV1 spec compatible.
 * Update svt.cmd/svt.sh/LocalSvt.cmake to v3.0.1.
+* Properly deal with grayscale PNG/JPG images in avifenc/avifdec
```

Comment:
This is a little vague if someone were only reading the changelog. Maybe just say that the conversion to RGB is skipped now?

Reply:
Done